eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -192,7 +193,7 @@ class Peer(val nodeParams: NodeParams, remoteNodeId: PublicKey, wallet: OnChainA
             stay()
         }
 
-      case Event(open: protocol.OpenDualFundedChannel, d: ConnectedData) =>
+        case Event(open: protocol.OpenDualFundedChannel, d: ConnectedData) =>
```

Comment:
nit: the previous formatting was correct, this one isn't?

Reply:
Must have slipped in somehow, fixed in [ae99140](https://github.com/ACINQ/eclair/pull/2552/commits/ae991400f9cf8c3ca40869abf69cc8141ab8363e)